app/rxjs-operators.ts

```diff
@@ -0,0 +1,15 @@
+// import 'rxjs/Rx'; // adds ALL RxJS statics & operators to Observable
+
+// See node_module/rxjs/Rxjs.js
+// Import just the rxjs statics and operators we need for THIS app.
```

Comment:
The Angular2 http get function uses Observables from the RxJS library. This file is used to include a global selection of operators. `import 'rxjs/Rx` would be fine too, as it includes the complete RxJS library, but this might blow up the application size (don't know how much of an issue this actually is, but the recommendation in the Angular2 Documentation is to import only a selection of operators, see [here](https://angular.io/docs/ts/latest/guide/server-communication.html#!#rxjs)). 

I fixed the import statements in 9dbbe44 so this file is actually used.